[2018-02-12 15:30:25.7364]ERROR
异常代码：{0x4c08a743,0x28db,0x47b8,{0x9b,0xa2,0x0a,0x31,0xe0,0x0d,0x87,0x94}}
Plugin.OAMessage.OAMessageException: 数据验证失败。调用OA接口出现异常 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 192.168.50.118:80
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.Web.Services.Protocols.SoapHttpClientProtocol.Invoke(String methodName, Object[] parameters)
   在 Plugin.OAMessage.OAMessage.WebServices.OfsTodoDataWebService.receiveRequestInfoByMap(anyType2anyTypeMapEntry[] in0) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\05.OAMessage\Web References\OAMessage.WebServices\Reference.cs:行号 165
   在 Plugin.OAMessage.OAMessageBuilder.CommonReceive(String flowid, String flowtitle, String workflowname, String nodename, String pcurl, String appurl, String creator, String receiver, Int32 isremark, Int32 viewtype, DateTime createtime) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\05.OAMessage\OAMessageBuilder.cs:行号 125
   --- 内部异常堆栈跟踪的结尾 ---
   在 Plugin.OAMessage.OAMessageBuilder.CommonReceive(String flowid, String flowtitle, String workflowname, String nodename, String pcurl, String appurl, String creator, String receiver, Int32 isremark, Int32 viewtype, DateTime createtime) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\05.OAMessage\OAMessageBuilder.cs:行号 159
   在 Plugin.OAMessage.OAMessageBuilder.ReceiveRead(String flowid, String nodename, String receiver) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\05.OAMessage\OAMessageBuilder.cs:行号 84
   在 Plugin.Workflow.WorkflowBuilder.ReadWorkflowProcess(String businessID, Employee lastUser) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\07.Workflow\WorkflowBuilder.cs:行号 14
   在 WebApplication.WebAPI.Controllers.TodosController.Load(String businessId) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TodosController.cs:行号 180
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
[2018-02-12 16:03:26.8773]ERROR
异常代码：{0x9e603737,0x50ce,0x42dd,{0x8c,0x30,0x71,0x74,0xff,0x86,0x7f,0x7b}}
System.Exception: 文件上传失败：找不到网络路径。
，   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   在 Plugin.Storage.NASStorageProvider.LoadFile(String fileAddress) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\06.Storage\NASStorageProvider.cs:行号 70
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 152
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 157
   在 Lib.BLL.Builder.TaskCollectionDataBuilder.BuildData(Boolean c1, Boolean c2, Boolean c3, Boolean c4, String& fileExt) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\04.BLL\Builder\TaskCollectionDataBuilder.cs:行号 32
   在 WebApplication.WebAPI.Controllers.TasksController.LoadSummaryData(String taskId, String corp, String org, String name, String username) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TasksController.cs:行号 337
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
[2018-02-12 16:05:03.5366]ERROR
异常代码：{0x86683b0c,0x77fb,0x4f35,{0xa0,0x8d,0x35,0x9c,0x4c,0xea,0x26,0xe3}}
System.Exception: 文件上传失败：找不到网络路径。
，   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   在 Plugin.Storage.NASStorageProvider.LoadFile(String fileAddress) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\06.Storage\NASStorageProvider.cs:行号 70
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 152
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 157
   在 Lib.BLL.Builder.TaskCollectionDataBuilder.BuildData(Boolean c1, Boolean c2, Boolean c3, Boolean c4, String& fileExt) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\04.BLL\Builder\TaskCollectionDataBuilder.cs:行号 32
   在 WebApplication.WebAPI.Controllers.TasksController.LoadSummaryData(String taskId, String corp, String org, String name, String username) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TasksController.cs:行号 337
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
[2018-02-12 16:05:03.5366]ERROR
异常代码：{0x1995e1b2,0xef08,0x4d65,{0xbb,0xe1,0x51,0x00,0x15,0x7d,0xd6,0xff}}
System.Exception: 文件上传失败：找不到网络路径。
，   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   在 Plugin.Storage.NASStorageProvider.LoadFile(String fileAddress) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\06.Storage\NASStorageProvider.cs:行号 70
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 152
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 157
   在 Lib.BLL.Builder.TaskCollectionDataBuilder.BuildData(Boolean c1, Boolean c2, Boolean c3, Boolean c4, String& fileExt) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\04.BLL\Builder\TaskCollectionDataBuilder.cs:行号 32
   在 WebApplication.WebAPI.Controllers.TasksController.LoadSummaryData(String taskId, String corp, String org, String name, String username) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TasksController.cs:行号 337
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
[2018-02-12 16:05:03.5366]ERROR
异常代码：{0x5dda5a89,0xcf83,0x4719,{0xae,0x11,0x48,0xe1,0x51,0xbd,0x94,0x1c}}
System.Exception: 文件上传失败：找不到网络路径。
，   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   在 Plugin.Storage.NASStorageProvider.LoadFile(String fileAddress) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\06.Storage\NASStorageProvider.cs:行号 70
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 152
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 157
   在 Lib.BLL.Builder.TaskCollectionDataBuilder.BuildData(Boolean c1, Boolean c2, Boolean c3, Boolean c4, String& fileExt) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\04.BLL\Builder\TaskCollectionDataBuilder.cs:行号 32
   在 WebApplication.WebAPI.Controllers.TasksController.LoadSummaryData(String taskId, String corp, String org, String name, String username) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TasksController.cs:行号 337
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
[2018-02-12 16:05:03.5747]ERROR
异常代码：{0x8dd0e321,0xf47a,0x40c3,{0x90,0x11,0x07,0x18,0x93,0xc3,0xce,0xcb}}
System.Exception: 文件上传失败：找不到网络路径。
，   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   在 Plugin.Storage.NASStorageProvider.LoadFile(String fileAddress) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\06.Plugin\06.Storage\NASStorageProvider.cs:行号 70
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 152
   在 Lib.Common.FileUploadHelper.DownLoadFileStream(String fileCode, Boolean isUseV1) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\00.Common\UploadHelper.cs:行号 157
   在 Lib.BLL.Builder.TaskCollectionDataBuilder.BuildData(Boolean c1, Boolean c2, Boolean c3, Boolean c4, String& fileExt) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\02.Lib\04.BLL\Builder\TaskCollectionDataBuilder.cs:行号 32
   在 WebApplication.WebAPI.Controllers.TasksController.LoadSummaryData(String taskId, String corp, String org, String name, String username) 位置 H:\work\LJTH\sjsj\code\LJTH\RCSJSJ\01.SourceCode\01.Solution\04.WebApplication\01.WebAPI\Controllers\TasksController.cs:行号 337
   在 lambda_method(Closure , Object , Object[] )
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
